assets/multiqc_config.yml

```diff
@@ -27,6 +27,14 @@ run_modules:
   - qualimap
   - kraken
 
+#skip_generalstats: true
+
+section_comments:
+  samtools: "WARNING! Those stats are calculated on the TRANSCRIPTOME BAM file, thus the inconsistency with STAR reported stats."
```

Comment:
please replace "thus the inconsistency with STAR reported stats" with "leading to possible inconsistencies STAR reported stats, which are calculated on the GENOME BAM file."